manager/common/license.go

```diff
@@ -120,8 +122,17 @@ type License struct {
 	Used  int
 }
 
-var totalCadenceLic = 3
-var totalXilinxLic = 3
+func strToInt(num string) int {
```

Comment:
You have two entirely orthogonal features in one PR: 1) to make the client timeout configurable, 2) to change the server. Create two separate PRs.